checks/limits.py

```diff
@@ -0,0 +1,22 @@
+from lib.base import CheckBase
+
+
+class CheckLimits(CheckBase):
+    whitelist = ['DeploymentConfig']
+
+    enable_parameter = 'limits'
+    description = 'Check that limits are defined'
+
+    def check_limits_cpu(self, m):
+        try:
+            containers = m['spec']['template']['spec']['containers']
+        except (KeyError, TypeError):
+            return
+
+        for c in containers:
+            try:
+                cpu_limits = c['resources']['limits']['cpu']
+            except (KeyError, TypeError):
+                cpu_limits = None
+
```

Comment:
Depending on the exact need, we'd need to ensure we have resource requests & limits for both cpu and memory

- `.resources.requests.cpu`
- `.resources.requests.memory`
- `.resources.limits.cpu`
- `.resources.limits.memory`

Reply:
Checking resource requests should probably go in it's own check module for more flexibility.

Reply:
done!

Reply:
absolutely! done :)